contract-tests/Cargo.toml

```diff
@@ -14,7 +14,7 @@ launchdarkly-server-sdk = { path = "../launchdarkly-server-sdk/", default-featur
 serde = { version = "1.0.132", features = ["derive"] }
 serde_json = "1.0.73"
 futures = "0.3.12"
-hyper = { version = "0.14.17", features = ["client"] }
+hyper = { version = "0.14.19", features = ["client"] }
```

Comment:
See https://github.com/launchdarkly/rust-eventsource-client/pull/72 for related changes to the event source.

Once the eventsource crate has been published, I'll update that dependency as well.